templates/telemetry/firefox_kpi_dashboard_v1/query.sql

```diff
@@ -61,7 +63,25 @@ WITH
           'GB',
           'CA'),
         mau,
-        0)) AS mau
+        0)) AS mau,
+    SUM(
+    IF
+      (country IN ('US',
+          'FR',
+          'DE',
+          'GB',
+          'CA'),
+        wau,
+        0)) AS wau,
```

Comment:
defining a query-scope UDF `udf_country_is_tier1` would remove the need to repeat this list so much